packages/scaffolding-cli/src/domain/config/static.config.json

```diff
@@ -199,8 +179,8 @@
     ]
   },
   "javaSpring": {
-    "gitRepo": "https://github.com/spring-guides/gs-actuator-service.git",
-    "gitRef": "17bbbb65c1c641634a59568c2e3642ae59e2e7dd",
+    "gitRepo": "https://github.com/amido/stacks-java.git",
+    "gitRef": "initial-commit",
```

Comment:
There isn't a master branch or init tag we could create?

Reply:
I think we should let the Java team control this going forward - @peet-j @ElvenSpellmaker

Reply:
Once we merge the initial-commit branch into master (hopefully today) I'll see about tagging a 0.0.1 initial release, we may wait until it's more fleshed out though. 🙂